dns/unbound-plus/src/opnsense/service/conf/actions.d/actions_unboundplus.conf

```diff
@@ -10,3 +10,10 @@ parameters:
 type:script
 message:fetching DNSBLs and restart Unbound
 description: Download Unbound DNSBLs and restart
+
+[unboundreload]
```

Comment:
I guess this should be a core command instead? As a side note, reloading will flush the cache as well.

Reply:
https://github.com/opnsense/core/commit/900b23a97b please use this one

Reply:
@fichtner there is no serivcecontroller for Unbound itself, shall I create one or do you want to take care of?

Reply:
I’ll do it, but not at the moment. Better to mock this here until we move Unbound to MVC itself. @AdSchellevis agreed that eventually we should release this plugin to core and maybe switch all of Unbound while at it. Unbound is already hybrid so we might as well push it through. 😊

Reply:
Good idea :) Shall I add existing features like overrides or advanced section to this plugin so you can cherrypick?

Reply:
let's only add fully functional parts for now